src/training.py

```diff
@@ -188,7 +191,7 @@ def train_model(
     train_losses = np.zeros(epochs)
     valid_losses = np.zeros(epochs)
 
-    pb = tqdm(total=epochs, desc="Epochs")
+    #pb = tqdm(total=epochs, desc="Epochs")
```

Comment:
I would just delete this line at this point.

Reply:
done